test/legacy_test/test_read_file.py

```diff
@@ -33,38 +35,50 @@ def setUp(self):
     def tearDown(self):
         self.temp_dir.cleanup()
 
-    def read_file_decode_jpeg(self):
+    def test_read_file_decode_jpeg_dynamic(self):
         if not paddle.is_compiled_with_cuda():
             return
-
         img_bytes = read_file(self.img_path)
-
         img = decode_jpeg(img_bytes, mode='gray')
         img = decode_jpeg(img_bytes, mode='rgb')
-
         img = decode_jpeg(img_bytes)
-
         img_cv2 = cv2.imread(self.img_path)
-        if paddle.in_dynamic_mode():
-            np.testing.assert_equal(img.shape, img_cv2.transpose(2, 0, 1).shape)
-        else:
-            place = paddle.CUDAPlace(0)
-            exe = paddle.static.Executor(place)
-            exe.run(paddle.static.default_startup_program())
-            out = exe.run(
-                paddle.static.default_main_program(), fetch_list=[img]
-            )
+        np.testing.assert_equal(img.shape, img_cv2.transpose(2, 0, 1).shape)
 
-            np.testing.assert_equal(
-                out[0].shape, img_cv2.transpose(2, 0, 1).shape
-            )
 
-    def test_read_file_decode_jpeg_dynamic(self):
-        self.read_file_decode_jpeg()
+class TestReadFileWithStatic(unittest.TestLoader):
```

Comment:
```suggestion
class TestReadFileWithStatic(unittest.TestCase):
```

设置错了的话单测不会运行，导致 ci-coverage 不通过

Reply:
收到